test/fixtures.js

```diff
@@ -1,18 +1,192 @@
+// Contact roles
+// 082111: registered health worker
+// 082222: unregistered person
+
 module.exports = function() {
 return [
 
+    // authentication
+        // get contact 082111 by msisdn (to validate msisdn check)
+        {
+            'request': {
+                'method': 'GET',
+                'params': {
+                    'details__addresses__msisdn': '+25682111'
+                },
+                'headers': {
+                    'Authorization': ['Token test_key'],
+                    'Content-Type': ['application/json']
+                },
+                'url': 'http://localhost:8001/api/v1/identities/search/',
+            },
+            'response': {
+                "code": 200,
+                "data": {
+                    "count": 1,
+                    "next": null,
+                    "previous": null,
+                    "results": [{
+                        "url": "http://localhost:8001/api/v1/identities/cb245673-aa41-4302-ac47-00000000001/",
+                        "id": "cb245673-aa41-4302-ac47-00000000001",
```

Comment:
Let's make the url and id end in `82111` here rather than `00001` so it's easier to find in the future. (similar for 82222, etc)